app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavScreen.kt

```diff
@@ -113,6 +116,7 @@ fun RootNavScreen(
         is RootNavState.VaultUnlockedForFido2Save,
         is RootNavState.VaultUnlockedForFido2Assertion,
         is RootNavState.VaultUnlockedForFido2GetCredentials,
+        is RootNavState.OnboardingStep,
```

Comment:
This seems wrong, Shouldn't this be the ONBOARDING_GRAPH?

Reply:
I'm agree, and I'm also curious about the naming...`OnboardingStep` sounds like a single step in the broader onboarding flow. Is there a reason not to just go with `Onboarding` for example?

Reply:
I am actually now thinking that onboarding flow nav can just be all state based of this value and therefore should be flattened to a nav state per "step". Thank y'all!

Reply:
Just to add context from chat in standup: desired behavior is that each onboarding step would be a top level root destination, i.e. user does system back and exits the app. So will need to have this be one state per onboard status and navigate to that screen as the root.